server_installation/topics/operator/keycloak-cr.adoc

```diff
@@ -0,0 +1,67 @@
+
+=== Keycloak Custom Resource
+
+The {project_operator} allows users to manage {project_name} clusters by using Keycloak Custom Resources:
+
+.`Keycloak` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: Keycloak
+metadata:
+  name: example-keycloak
+  labels:
+    app: sso
+spec:
+  instances: 1
+  externalAccess:
+    enabled: True
+```
+
+The `Spec` contains three properties (two of them are listed the above and the external database has been covered in the other paragraphs of this manual):
```

Comment:
@slaskawi `s/two of them are listed the above/two of them are listed above/g`

Reply:
Fixed.